addons/mail/static/src/discuss/call/common/call_action_list.js

```diff
@@ -15,9 +18,14 @@ export class CallActionList extends Component {
 
     setup() {
         super.setup();
+        this.INDICATOR = INDICATOR;
         this.store = useService("mail.store");
         this.rtc = useService("discuss.rtc");
         this.callActions = useCallActions();
+        this.more = useRef("more");
+        this.popover = usePopover(Tooltip, {
+            position: 'top-middle',
+        });
```

Comment:
Make sure linter is properly set up on you machine: it should have converted the `'` to `"` there.